nautilus_trader/common/actor.pxd

```diff
@@ -251,14 +257,15 @@ cdef class Actor(Component):
         list bar_types,
         datetime start=*,
         datetime end=*,
+        int limit=*,
         ClientId client_id=*,
         callback=*,
         bint include_external_data=*,
         bint update_subscriptions=*,
         bint update_catalog=*,
         dict[str, object] params=*,
     )
-    cpdef bint is_pending_request(self, UUID4 request_id)
+    cpdef bint is_pending_request(self, UUID4 id)
```

Comment:
I think we should leave the param name as `request_id`, as `id` shadows the built-in and isn't as explicit.

[Also applies to many changes below]